libmamba/CMakeLists.txt

```diff
@@ -76,6 +76,11 @@ if(BUILD_STATIC)
     add_definitions(-DLIBMAMBA_STATIC_DEPS)
 endif()
 
+# Use mamba installation prefix to set root prefix (base)
+if(BUILD_SHARED OR BUILD_MAMBA)
+    add_definitions(-DMAMBA_USE_INSTALL_PREFIX_AS_BASE)
+endif()
```

Comment:
Thinking again about this. I think the definition should be passed to the target for the shared library, and not for the static library.

What do you think?